go.mod

```diff
@@ -69,8 +69,8 @@ require (
 	github.com/smartcontractkit/chainlink-relay v0.1.7-0.20230814175909-17597db0d6fc
 	github.com/smartcontractkit/chainlink-solana v1.0.3-0.20230802143301-165000751a85
 	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.1-beta-test.0.20230802150127-d2c95679d61a
-	github.com/smartcontractkit/libocr v0.0.0-20230816220705-665e93233ae5
-	github.com/smartcontractkit/ocr2keepers v0.7.10
+	github.com/smartcontractkit/libocr v0.0.0-20230802221916-2271752fa829
```

Comment:
Is this stale? `665e93233ae5` is head of master while `2271752fa829` is a few version back.

Reply:
not sure how this was updated. will revert